components/benefit_markets/app/models/benefit_markets/services/pvp_eligibility_service.rb

```diff
@@ -16,12 +16,16 @@ def create_or_update_pvp_eligibilities
 
         @args[:rating_areas].each do |rating_area_id, evidence_value|
           pvp = find_or_create_pvp(rating_area_id)
-          effective_date = @args[:effective_date] || TimeKeeper.date_of_record
-          current_eligibility = pvp.latest_active_pvp_eligibility_on(effective_date)
-          next if current_eligibility.present?.to_s == evidence_value.to_s
-
-          result = store_pvp_eligibility(pvp, evidence_value, effective_date)
-          eligibility_result[rating_area_id] = result.success? ? "Success" : "Failure"
+          if pvp.present?
+            current_eligibility = pvp.latest_active_pvp_eligibility_on(TimeKeeper.date_of_record)
+            next if current_eligibility.present?.to_s == evidence_value.to_s
+
+            effective_date = get_effective_date(evidence_value)
+            result = store_pvp_eligibility(pvp, evidence_value, effective_date)
+            eligibility_result[rating_area_id] = result.success? ? "Success" : "Failure"
+          else
+            eligibility_result[rating_area_id] = "Failure"
+          end
         end
 
         grouped_eligibilities = eligibility_result.group_by { |_year, value| value }
```

Comment:
If you do end up making edits in this file, maybe we can update the likely incorrect `_year` param here to head off (minor) confusion.